daemon/cmd/kube_proxy_replacement.go

```diff
@@ -339,6 +340,10 @@ func initKubeProxyReplacementOptions() (strict bool) {
 
 // handleNativeDevices tries to detect bpf_host devices (if needed).
 func handleNativeDevices(strict bool) {
+	if option.Config.EnableWireguard {
+		option.Config.DirectRoutingDevice = wireguard.IfaceName
```

Comment:
Maybe it's worth extracting common defines into a `types` package within `pkg/wireguard`, as to avoid importing the entire `agent` package just for a define